<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filesSum</class>
 <widget class="QWidget" name="filesSum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_choose">
       <property name="text">
        <string>choose file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_filename"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_result"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="rbtn_crc16">
       <property name="text">
        <string>crc16</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_sum</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_md5">
       <property name="text">
        <string>md5sum</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_sum</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_sha1">
       <property name="text">
        <string>sha1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_sum</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_sha256">
       <property name="text">
        <string>sha256</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_sum</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>exit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_calc">
       <property name="text">
        <string>calc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rbtnGroup_sum"/>
 </buttongroups>
</ui>
